tests/bluetooth/audio/ascs/src/main.c

```diff
@@ -500,6 +500,11 @@ ZTEST_F(ascs_test_suite, test_cis_link_loss_in_streaming_state)
 
 	/* Expected to notify the upper layers */
 	expect_bt_bap_stream_ops_qos_set_called_once(stream);
+	if (IS_ENABLED(CONFIG_BT_ASCS_ASE_SNK)) {
+		expect_bt_bap_stream_ops_disabled_called_once(stream);
+	} else {
+		expect_bt_bap_stream_ops_disabled_not_called();
+	}
```

Comment:
This seems incorrect. The change is specifically to ensure that sink and source ASEs behave the same with regard to the `disabled` callback.